client/Tutorial/TutorialSteps.ts

```diff
@@ -82,51 +152,132 @@ export const TutorialSteps: TutorialStep[] = [
+      return { left, top };
+    },
+    HighlightSelector: ".active-combatant",
   },
+  // step 11 - select combatant
   {
     Message:
-      "Select a combatant by clicking. You can select multiple combatants by holding the control key.",
+      "Select a combatant by <strong>clicking</strong> on a combatant. You can select multiple combatants by <strong>holding the control key</strong>.",
     RaiseSelector: ".combatants, .right-column",
+    AwaitAction: "combatant-click",
     CalculatePosition: elements => {
       const element = elements[0];
       const location = getLocation(element);
       const left = location.left + 5;
       const top = location.top + location.height + 10;
       return { left, top };
-    }
+    },
+    HighlightSelector: ".selected-combatant, .combatant",
+  },
+  // step 12 - view combatant action menu
+  {
+    Message:
+      "On the left side of the screen, you can see the <strong>combatant action menu</strong>.",
+    RaiseSelector: ".commands-combatant",
+    CalculatePosition: elements => {
+      const element = elements[0];
+      const location = getLocation(element);
+      const left = location.left + 5;
+      const top = location.top + location.height - 525;
+      return { left, top };
+    },
+    HighlightSelector: ".commands-combatant, .scrollframe",
   },
+  // step 13 - apply damage
   {
     Message:
-      "Press 't' or click 'Apply Damage' to apply damage to selected combatants. You can enter a negative number to apply healing.",
-    RaiseSelector: ".combatants, .c-button--apply-damage, .prompts, .prompt",
+      "let's continue and apply some damage to a selected combatant.<strong>Click 'Apply Damage'</strong> to apply damage to selected combatants. You can enter a <strong>negative number</strong> to apply healing.",
+    RaiseSelector: ".c-button--apply-damage, .combatants,  .prompts, .prompt",
     AwaitAction: "ApplyDamage",
     CalculatePosition: elements => {
       const element = elements[0];
       const location = getLocation(element);
       const left = location.left + location.width + 10;
       const top = location.top + 5;
       return { left, top };
-    }
+    },
+    HighlightSelector: ".c-button--apply-damage, .pronmpt",
   },
-  /*{
-        Message: "Press 'n' or click 'Next Turn' to advance the tracker. The active combatant's statblock is displayed for convenience.",
-        RaiseSelector: ".c-button--next-turn, .left-column, .combatants",
-        CalculatePosition: elements => {
-            const element = elements.first();
-            const left = location.left + element.outerWidth() + 10;
-            const top = location.top + 5;
-            return { left, top };
-        }
-    },*/
+  // step 14 - view remove from encounter button
+  {
+    Message:
+      "The next button in the combatant action menu is the <strong>Remove from Encounter</strong> button. This button will remove the <strong>selected combatants</strong> from the encounter.",
```

Comment:
Instead of 'The next button', you could just write 'This is the ...' since the UI is already drawing their attention here.